nca/Utils/CmdlineRunner.py

```diff
@@ -88,15 +88,16 @@ def locate_kube_config_file():
         raise FileNotFoundError('Failed to locate Kubernetes configuration files')
 
     @staticmethod
-    def get_k8s_resources(resource):
+    def get_k8s_resources(resources):
         """
         Run kubectl to get the list of available instances of a given resource
-        :param str resource: The name of the resource
+        :param list resources: The list of resource names
         :return: The output of 'kubectl get' (should be a list-resource)
         """
         CmdlineRunner.locate_kube_config_file()
-        cmdline_list = ['kubectl', 'get', resource, '-o=json']
-        if resource in ['networkPolicy', 'authorizationPolicy', 'pod', 'ingress', 'Gateway', 'VirtualService', 'sidecar']:
+        cmdline_list = ['kubectl', 'get', ','.join([r for r in resources]), '-o=json']
+        if set(['networkPolicy', 'authorizationPolicy', 'pod', 'ingress', 'Gateway', 'VirtualService', 'sidecar',
+                'service', 'serviceentry']).intersection(resources):
             cmdline_list.append('--all-namespaces')
```

Comment:
why appending `--all-namespaces` only if the intersection is not empty?

Reply:
Removed the condition